src/library/Uno.Material/Styles/Controls/ComboBox.xaml

```diff
@@ -285,7 +286,7 @@
 						<!-- Elevated View -->
 						<toolkit:ElevatedView Background="{ThemeResource ComboBoxBackgroundColorBrush}"
 											  HorizontalAlignment="Stretch"
-											  CornerRadius="4"
+											  CornerRadius="{StaticResource DefaultComboBoxBorderRadius}"
```

Comment:
Shouldn't the variable name start with `ComboBox` like on all variables on zeplin?

`ComboBoxBorderCorderRadius` ?

Reply:
Done